Sources/Sentry/SentryAppStartMeasurement.m

```diff
@@ -41,8 +60,50 @@ - (instancetype)initWithType:(SentryAppStartType)type
     }
 
     return self;
+#    else
+    SENTRY_LOG_DEBUG(@"SentryAppStartMeasurement only works with UIKit enabled. Ensure you're "
+                     @"using the right configuration of Sentry that links UIKit.");
+    return nil;
+#    endif // SENTRY_HAS_UIKIT
+}
+
+#    if SENTRY_HAS_UIKIT
+- (SentryAppStartType)type
+{
+    return _type;
+}
+
+- (BOOL)isPreWarmed
+{
+    return _isPreWarmed;
+}
+
+- (NSTimeInterval)duration
+{
+    return _duration;
+}
+
+- (NSDate *)appStartTimestamp
+{
+    return _appStartTimestamp;
+}
+
+- (NSDate *)runtimeInitTimestamp
+{
+    return _runtimeInitTimestamp;
+}
+
+- (NSDate *)moduleInitializationTimestamp
+{
+    return _moduleInitializationTimestamp;
+}
+
+- (NSDate *)didFinishLaunchingTimestamp
+{
+    return _didFinishLaunchingTimestamp;
 }
+#    endif // SENTRY_HAS_UIKIT
```

Comment:
`l`: Do we need these getters?

Reply:
nope, you're right. forgot that i'd originally added them to override depending on the macro, but then realized that if i make the inits return nil, no overrides were necessary. they're gone!